src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoUtilsTest.java

```diff
@@ -22,6 +25,37 @@
 public class DynamoUtilsTest {
 
     @Test
+    public void testTableName() {
+        final Config config = new Config() {
+            @Override
+            public String getUser() {
+                return "testTableName";
+            }
+            @Override
+            public Environment getEnvironment() {
+                return Environment.LOCAL;
+            }
```

Comment:
Would it be better to use a mock for this? At the very least, we'd have to write less code.